modules/configuration/src/main/java/org/apache/ignite/internal/configuration/notifications/ConfigurationNotificationEventImpl.java

```diff
@@ -15,47 +15,50 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.internal.configuration.util;
+package org.apache.ignite.internal.configuration.notifications;
 
-import java.util.Map;
 import org.apache.ignite.configuration.ConfigurationProperty;
 import org.apache.ignite.configuration.notifications.ConfigurationNotificationEvent;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Implementation of the {@link ConfigurationNotificationEvent}.
+ *
+ * @param <VIEWT> Type of the subtree or the value that has been changed.
  */
 class ConfigurationNotificationEventImpl<VIEWT> implements ConfigurationNotificationEvent<VIEWT> {
-    /** Previous value of the updated configuration. */
+    /** Old value. */
```

Comment:
Is it really better than the old comment?

Reply:
Not better, returned the previous description.